src/v2/components/TourDeSol/Cards/index.jsx

```diff
@@ -41,19 +43,19 @@ const Cards = ({
     },
     {
       title: 'Total SOL In Circulation',
-      value: (cluster.supply / Math.pow(2, 34)).toFixed(2),
+      value: (supply / Math.pow(2, 34)).toFixed(2),
       changes: '',
       period: 'since yesterday',
     },
     {
       title: 'Total Staked Tokens',
```

Comment:
"Total Staked SOL" (Can we sneak this in? I believe @manuel-calavera updated the value, but we need to edit the title too)